website/docs/cli/config/config-file.html.md

```diff
@@ -287,6 +287,10 @@ the directories indicated above that exists when Terraform starts up.
 In addition, if a `terraform.d/plugins` directory exists in the current working
 directory, it will be added as a filesystem mirror.
 
+Note that providers installed in these directories need to be placed in the
```

Comment:
Nice! I like this extra note to make it more explicit.

I wonder about possibly connecting this more directly with the previous paragraph, which already talks about `filesystem_mirror` blocks, by reordering this info a little bit:

> If a directory `terraform.d/plugins` exists in the current working directory then Terraform will also include that directory, regardless of your operating system.
>
> Terraform will check each of the paths above to see if it exists, and if so treat it as a filesystem mirror. The directory structure inside each one must therefore match one of the two structures described for [`filesystem_mirror` blocks](#filesystem_mirror).

My intent here is to try to connect the idea that this is an implied `filesystem_mirror` block with the explicit note about the directory structure, to hopefully implicitly answer the question "why is this so?" at the same time.

Reply:
that's much nicer, thank you - pushed!